crates/primitives-traits/src/block/body.rs

```diff
@@ -28,10 +27,15 @@ pub trait BlockBody:
     + InMemorySize
     + MaybeSerde
 {
+    /// Uncle block header.
+    type Header: BlockHeader + 'static;
+
```

Comment:
this does not belong in this pr

Reply:
if I restrict the bound to just `'static` is that ok? that way the `Compact` trait bound can properly be applied on `FullBlockBody`.

Reply:
unclear why this is introduced, seems out of scope for this pr